testing/ember-app/tests/utils/function/state-test.ts

```diff
@@ -11,7 +11,7 @@ module('Utils | State | js', function (hooks) {
   setupTest(hooks);
 
   test('initial state', async function (assert) {
-    let m;
+    let m = '<invalid state>';
```

Comment:
Here `m` is used entirely to make test more readable by making the custom assertion message appear on its own line, rather than after the assertion where it's hard to notice and to distinguish from other arguments.